lib/nerves/artifact/resolvers/github_api.ex

```diff
@@ -44,4 +47,48 @@ defmodule Nerves.Artifact.Resolvers.GithubAPI do
       %{"url" => url} -> {:ok, url}
     end
   end
+
+  defp get_username(opts) do
+    case Keyword.fetch(opts, :username) do
+      {:ok, nil} ->
```

Comment:
Connor is speaking about this not matching. Should be just `:error`

Reply:
What if someone explicitly sets the option to `nil`? We might want both.